HW6/Dockerfile

```diff
@@ -0,0 +1,29 @@
+
+FROM python:3.6-buster
+#Not using this for this assignment but I think
+#I will need it for the final so I left it
+#buster was required to get the mariadb library installed
+#should probably remove
+
+RUN apt-get update \
+    && apt-get -yy install libmariadb-dev
+
```

Comment:
not sure what libmariadb-dev is but it seems like you also install the mariadb client below which is good from what I understand